StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -474,3 +474,32 @@ extension PaymentElementConfiguration {
         (defaultBillingDetails.email?.isEmpty == false && billingDetailsCollectionConfiguration.attachDefaultsToPaymentMethod)
     }
 }
+
+extension PaymentSheet.PaymentMethodType {
+    
+    var isLinkBankPayment: Bool {
+        switch self {
+        case .stripe:
+            return false
+        case .external:
+            return false
+        case .instantDebits:
+            return true
+        case .linkCardBrand:
+            return true
+        }
```

Comment:
Minor refactor suggestions:

```suggestion
        [.instantDebits, .linkCardBrand].contains(self)
```